store/src/main/java/com/nytimes/android/external/store/base/impl/RealInternalStore.java

```diff
@@ -227,6 +231,11 @@ void notifySubscribers(Parsed data) {
         return stream;
     }
 
+    @NonNull @Override
+    public Observable<Parsed> stream() {
+        return subject.asObservable();
```

Comment:
Observable<Foo> fooObservable = store.stream();

barObservable= fooObservable.map(foo->foo.bar());

barObservable.subscribe()
barObservable.subscribe()

map gets called twice.  I'd prefer for it to be called once